frame/scheduler/src/lib.rs

```diff
@@ -260,6 +262,55 @@ decl_module! {
 			Self::do_cancel_named(Some(origin.caller().clone()), id)?;
 		}
 
+		/// Anonymously schedule a task after a delay.
+		///
+		/// # <weight>
+		/// - S = Number of already scheduled calls
+		/// - Base Weight: 22.29 + .126 * S µs
```

Comment:
is this coming from a benchmark on the standard machine? this looks quite similar to `schedule_named ` to me, wondering why the weights differ.